src/main/resources/regression/miracl-v1.0-ar.yaml

```diff
@@ -0,0 +1,48 @@
+---
+corpus: miracl-v1.0-ar
+corpus_path: collections/miracl-corpus/miracl-corpus-v1.0-ar
+
+index_path: indexes/lucene-index.miracl-v1.0-arabic/
```

Comment:
Change to `lucene-index.miracl-v1.0-ar` to be consistent with corpus? i.e. two-letter ISO.